backend/fms_core/viewsets/project.py

```diff
@@ -24,6 +26,15 @@ class ProjectViewSet(viewsets.ModelViewSet, TemplateActionsMixin):
         **_project_filterset_fields,
     }
 
+    template_action_list = [
+        {
+            "name": "Link Projects with Samples",
+            "description": "Upload the provided template with up to 384 links between projects and samples.",
```

Comment:
Not sure we have a limit here ... i would remove the "up to 384" from the description.